app/views/admin/organizations/edit.html.erb

```diff
@@ -1,5 +1,5 @@
 <section class="content-header">
-<% content_for :title, "Edit - Organizations - #{current_organization.name}" %>
+<% content_for :title, "Edit - Organizations - {#@organization.name}" %>
```

Comment:
Is this a typo?

Reply:
yes, I didn't realize, thanks for telling me